rrfs-test/validated_yamls/templates/obtype_config/msonet_stationPressure_188.yaml

```diff
@@ -41,6 +41,13 @@
          # ------------------
          # stationPressure (188)
          # ------------------
+         # Accept, reject, or passivate (monitor)
+         - filter: Perform Action
+           filter variables:
+           - name: stationPressure
+           action:
+             name: accept # accept, reject, passivate
+
```

Comment:
I would suggest we comment out all monitor block be default.
Although the current form does not affect run results, but it takes extra time to loop through all observations in the obs space to mark the "accept" flag, which can be avoided, especially when there are lots of observations.

Reply:
Thanks for the suggestion and it is good to test this. 

I just ran two tests with the workflow (repeating the same 00Z cycle): one with all prepbufr obs set to "accept" and one with each of those "accept blocks" commented out. The run times were basically identical (within 3 seconds), so it doesn't seem to add any extra time to keep them uncommented--this difference is within the usual variance of run times for repeated runs.

Leaving them uncommented does make it easier to toggle on and off, since you only need to change a single line.

I'm not sure how this applies to radiance DA. That might need additional testing.

Reply:
Thanks for testing the timing.
Although it is not preferred to do it this way, there is no evident harm either. Thanks!